content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,751 @@
+---
+title: Ciclo de vida de un Pod
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+Esta página describe el ciclo de vida de un Pod.
+Los Pods siguen un ciclo de vida definido, comenzando en la fase [`Pending`](#pod-phase),
+y luego pasando a "en ejecución" `Running` si al menos uno de sus contenedores primarios se
+inicia correctamente,
+y luego pasando a "exitoso" (`Succeeded`) o "fallido" (`Failed`) si uno de los contenedores de un Pod
+termina en error.
+
+Al igual que contenedores de aplicaciones, los Pods se consideran entidades relativamente efímeras.
+Los Pods se crean y se les asigna un identificador único ([UID](/docs/concepts/overview/working-with-objects/names/#uids)), y se programan para ejecutarse en nodos donde se mantienen hasta que se terminan (de acuerdo con las políticas de reinicio) o se eliminan.
+
+Si un {{< glossary_tooltip text="nodo" term_id="node" >}} muere,
+los Pods programados para ejecutarse en ese Nodo
+se [programan para eliminarse](#pod-garbage-collection).
+El plano de control marca los Pods para ser eliminados luego de un periodo de tiempo.
+
+<!-- body -->
+
+## Ciclo de vida de un Pod
+
+Mientras un Pod se está ejecutando, el kubelet puede reiniciar contenedores para manejar algunos tipos de fallos.
+Dentro de un Pod, Kubernetes rastrea distintos [estados](#container-states) del contenedor y determina qué acción realizar para que el Pod esté sano nuevamente.
```

Comment:
This Hugo reference is missing. See the below suggestion: https://github.com/kubernetes/website/pull/47204/files#r1725984167